packages/fxa-auth-server/lib/senders/email.js

```diff
@@ -1936,6 +1935,7 @@ module.exports = function(log, config, oauthdb) {
       templateName,
       'account-settings'
     );
+    links.accountSettingsLinkAttributes = `href="${links.accountSettingsUrl}" target="_blank" style="color:#ffffff;font-weight:500;"`;
```

Comment:
We should probably add `rel="noopener noreferrer"`. I did a quick Google search ([see second answer](https://stackoverflow.com/questions/51193862/mailto-in-href-should-i-add-rel-noopener-noreferrer)) around how this affects mail clients and it looks like someone could be susceptible to attack if the receiver opens it in webmail. At best it offers protection and at worst it does nothing. 🤷‍♀️

Reply:
Doesn't look like we have this in any other links in email, might be worth a follow up issue?